adapters/repos/db/vector/flat/index_test.go

```diff
@@ -60,12 +66,27 @@ func run(dirName string, logger *logrus.Logger, compression string,
 		return 0, 0, err
 	}
 
+	pq := flatent.CompressionUserConfig{
+		Enabled: false,
+	}
+	bq := flatent.CompressionUserConfig{
+		Enabled: false,
+	}
+	switch compression {
+	case CompressionPQ:
+		pq.Enabled = true
+		pq.Rescore = 100 * k
+	case CompressionBQ:
+		bq.Enabled = true
+		bq.Rescore = 100 * k
+	}
 	index, err := flat.New(flat.Config{
 		ID:               runId,
 		DistanceProvider: distancer,
 	}, flatent.UserConfig{
-		Compression: compression,
-		EF:          100 * k,
+		VectorCache: false,
```

Comment:
shouldn't `VectorCache` be  assigned to arg `variableCache` instead of hardcoded `false`?